### ISSUE TITLE
any chance for a huion h610pro v2 compatibility?

### ISSUE THREAD

**Issue by Qzmikhailov** [hawku/TabletDriver#1174]
## Hardware/software information
Windows Version: 

Tablet Driver version: 

Tablet model: huion h610 pro v2

**Comment by SnowLire** [comment]
Nope. Nothing has been updated for the past 3-4 years and it won't change. Refer to #1087
